pkg/operator/ceph/controller/spec.go

```diff
@@ -692,6 +692,13 @@ func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Contai
 			size = 1
 			logger.Info("maxLogSize is 0M setting to minimum of 1M")
 		}
+
+		if strings.Contains(daemonID, "*-client.rbd-mirror*") && size < 1000 {
```

Comment:
I feel like it is a slippery slope to start giving individual daemons custom settings. I understand that practically, we may sometimes need to make certain provisions, but I don't think this has to be one of those times. Any time there are special provisions in the code, we are more likely to have to document those provisions or explain to some subset of users why there is a special behavior change. 

I would suggest that we keep the log collector simpler and rotate all ceph logs more frequently as Ilya suggested.

We might also want to consider whether we can keep the current log collector behavior and instead recommend changes to rbd-mirror daemon configuration to log less in environments where they are using too much space.

Reply:
> I understand that practically, we may sometimes need to make certain provisions, but I don't think this has to be one of those times.

The justification here is that rbd-mirror daemon is only just now starting to get some serious use and we need to be able to root cause some fairly complex teething issues.  I get that this may not be apparent on the Rook side since technically rbd-mirror daemon has been around for years but this is exactly the case for such a special provision IMO.  When rbd-mirror daemon becomes more robust and we no longer get much value from verbose logs that go back in time, the special provision can be taken out.

> I would suggest that we keep the log collector simpler and rotate all ceph logs more frequently as Ilya suggested.

My (rbd-mirror daemon specific) ask here is not to rotate more frequently but to keep a lot more rotations around.  If the desire is to expand the scope to all Ceph daemons, we may need to explicitly tell users to reserve/keep in mind a certain amount of space for logs.

> and instead recommend changes to rbd-mirror daemon configuration to log less

I'm afraid this is not option in the rbd-mirror daemon case.  In fact, we have just landed changes to make it log more for reasons stated above ;)